composer.json

```diff
@@ -1,13 +1,13 @@
 {
     "name": "optimus/api-consumer",
     "require": {
-        "laravel/framework": "~5.2"
+        "laravel/framework": "~6.0"
```

Comment:
Support for Laravel 6. 
Given the rest of the dependencies is either 5 or 6, so this new version would be support for Laravel 6 only